release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -41,17 +41,19 @@ This release adds improvements related to the following components and concepts.
 === {op-system-first}
 
 [id="ocp-4-7-enhanced-disk-provisioning"]
-==== Enhanced disk provisioning for LUKS and RAID devices
+==== Enhanced disk provisioning for LUKS, RAID, and FBA DASD
 
-{op-system} 4.7 includes several improvements to disk provisioning for bare metal deployments:
+{product-title} 4.7 includes several improvements to disk provisioning for bare metal deployments. The following features are currently supported for new 4.7 clusters only:
```

Comment:
Is this a case where we would use {product-version}? (Not being rhetorical, I'm not sure :slightly_smiling_face: )

Reply:
Not in this case, where I want the specific version (4.7) to be hardcoded because it is likely to be different in 4.8.